### PR TITLE
Further improve test coverage for ReadCSV

### PR DESCRIPTION
**Pull request by alex-konovalov**
I was going to edit #2535 to add one more test as @hulpke suggested in #2535, but it was merged in the meantime by @fingolfin, so I am doing this in a separate PR.

### CODE REVIEW

File: tst/testinstall/strings.tst [hunk 1]
```diff
@@ -262,6 +262,12 @@ gap> fname := Filename(tmpdir, "output.csv");;
 gap> PrintCSV( fname, s );
 gap> s = ReadCSV( fname );
 true
+gap> s := ReadCSV( fname, true );
```
Comment (fingolfin) [inline tst/testinstall/strings.tst]: This new test actually does not use RCSVSplitString. To do that, the input must contain quotes " and use commas as sepetators. The quotes then are used to mask fields containing commas.

Comment (hulpke) [reply tst/testinstall/strings.tst]: @fingolfin 
You're right. We need to force commas in strings, and maybe lets add some gratuitous quotes. But it does not need to be the mask.

What about

    gap> s[2].emil:="\"Alas, poor Yorick\", the call went";;
    gap> PrintCSV(fnam,s);
    gap> ReadCSV(fnam,true);
    [ rec( field1 := "emil", field2 := "f", field3 := "f1", field4 := "f 2" ),
      rec( field2 := 2, field3 := 1, field4 := 3 ),
      rec( field1 := "\"Alas, poor Yorick\", the call went", field3 := 4, field4 := 6 ),
      rec( field3 := 7 ),  rec( field2 := 11, field4 := 12 ),
      rec( field2 := "yy", field3 := "x", field4 := "zzz" ) ]

**Comment by alex-konovalov** [comment]
Pushed "poor Yorick" example - let's see...

**Comment by alex-konovalov** [comment]
Ok, now https://codecov.io/gh/gap-system/gap/pull/2537/changes shows 6.3% increase in `string.gi` coverage, thanks @hulpke for the tip!

File: tst/testinstall/strings.tst [hunk 2]
```diff
@@ -262,6 +262,14 @@ gap> fname := Filename(tmpdir, "output.csv");;
...
 gap> s = ReadCSV( fname );
 true
+gap> s[2].emil:="\"Alas, poor Yorick\", the call went";;
+gap> PrintCSV(fname,s);
+gap> ReadCSV(fname,true);
+[ rec( field1 := "emil", field2 := "f", field3 := "f1", field4 := "f 2" ), 
+  rec( field2 := 2, field3 := 1, field4 := 3 ), 
+  rec( field1 := "\"Alas, poor Yorick\", the call went", field3 := 4, 
+      field4 := 6 ), rec( field3 := 7 ), rec( field2 := 11, field4 := 12 ), 
+  rec( field2 := "yy", field3 := "x", field4 := "zzz" ) ]
```
Comment (fingolfin) [inline tst/testinstall/strings.tst]: Can you please also add an "empty" field? I.e. containing an empty string.

Comment (alex-konovalov) [reply tst/testinstall/strings.tst]: Done

**Pull request merged**